CMakeLists.txt

```diff
@@ -221,3 +221,5 @@ if(packaging)
 		)
 	endif()
 endif()
+
+add_subdirectory(${3RDPARTY_DIR}/CLI11)
```

Comment:
This should be added, before we let cmake run down the subdirectories that actually make use of this - aka: before `add_subdirectory(src)`